src/components/ContainerSettingsAdvanced.react.js

```diff
@@ -1,8 +1,12 @@
+import _ from 'underscore';
 import React from 'react/addons';
 import metrics from '../utils/MetricsUtil';
 import ContainerUtil from '../utils/ContainerUtil';
 import containerActions from '../actions/ContainerActions';
 
+var ipRegex = /\b(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\b/;
+var domainRegex = /([a-z0-9-]+\.(?:com|net|org|co\.uk|localhost|dev))(?:\/|$)/;
```

Comment:
I think this is overly restrictive. The web is not limited to just `.com`,  `.net`, `.org`, `.co.uk`, `.localhost` and `.dev`.
Maybe something like `/^([a-z0-9-]+(?:\.[a-z0-9-]+)+)(?:\/|$)/` is be better.